accountsdb-plugin-postgres/src/transaction_selector.rs

```diff
@@ -0,0 +1,185 @@
+/// The transaction selector is responsible for filtering transactions
+/// in the plugin framework.
+use {log::*, solana_sdk::pubkey::Pubkey, std::collections::HashSet};
+
+pub(crate) struct TransactionSelector {
+    pub mentions: HashSet<Vec<u8>>,
```

Comment:
nit: `mentions` -> `selected_account_keys`

Reply:
This is referring to the existing pub enum TransactionLogCollectorFilter {
    All,
    AllWithVotes,
    None,
    OnlyMentionedAddresses,
}